src/integrationTest/java/org/opensearch/test/framework/TestSecurityConfig.java

```diff
@@ -252,7 +256,7 @@ public XContentBuilder toXContent(XContentBuilder xContentBuilder, Params params
         }
     }
 
-    public static class User implements UserCredentialsHolder, ToXContentObject {
+    public static class User implements UserCredentialsHolder, ToXContentObject, Serializable, Writeable {
```

Comment:
Out of curiosity, why do we need to have the user class as Serializable?

Reply:
Removed